docs/src/views/FAQ.tsx

```diff
@@ -111,6 +111,26 @@ export default () => (
       }
     />
 
+    <Question content="What's the difference between Flex, Grid, Layout, Box and Segment components?" />
+    <Answer
+      content={
+        <p>
+          Flex, Grid and Layout components handle layout aspects. The Flex component is for laying
```

Comment:
nit: as we are doing throughout the docs, lets wrap component names in `code('Flex')`, so that it would lead to proper formatting (just as an example, here is the first paragraph from the `Layout` section): 

![image](https://user-images.githubusercontent.com/9024564/61526988-bdffaf00-aa1b-11e9-8af8-f423ac8286a7.png)